docker/vast/docker-compose.yaml

```diff
@@ -0,0 +1,17 @@
+version: "3"
+services:
+  vast:
+    container_name: tenzir-vast
```

Comment:
why not just `vast`? why defining the container name at all?

Reply:
It's because we need a unique prefix for all services in the test script. I've fixed this by setting `COMPOSE_PROJECT_NAME` in the test script locally.